src/open_samus_returns_rando/specific_patches/cosmetic_patches.py

```diff
@@ -48,3 +50,37 @@ def music_shuffle(editor: PatcherEditor, configuration: dict) -> None:
         original_track = f"sounds/streams/music/{original}.bcwav"
         new_track = temp_dict[f"sounds/streams/music/{new}.bcwav"]
         editor.replace_asset(original_track, new_track)
+
+
+def volume_patches(editor: PatcherEditor, configuration: dict) -> None:
+    music = configuration["volume_adjustments"]["music"]
+    environment_sfx = configuration["volume_adjustments"]["environment_sfx"]
+    # If there are no custom values, quit
+    if music == 1 and environment_sfx == 1:
+        return
+
+    # Music Adjustments
+    sound_defs = editor.get_file("system/snd/scenariomusicdefs.bmdefs", Bmdefs)
+    sounds = sound_defs.raw["sounds"]
+    for sound in sounds:
+        # Apply the music values to each track's music volume
+        sound["volume"] *= music
+
+    enemies_list = sound_defs.raw["enemies_list"]
+    for enemy in enemies_list:
+        areas = enemy["areas"]
+        for area in areas:
+            layers = area["layers"]
+            for layer in layers:
+                states = layer["states"]
+                for state in states:
```

Comment:
why not just do this
```suggestion
    for enemy in enemies_list:
        for area in enemy["areas"]:
            for layer in area["layers"]:
                for state in layer["states"]:
```
You're never using the other variables anyway.